src/containers/shared/components/DomainLink.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { decodeHex } from '../transactionUtils'
+
+interface Props {
+  className?: string
+  decode?: boolean
+  domain: string
+}
+
+const DomainLink = (props: Props) => {
+  const { className, decode, domain } = props
+
+  return (
+    <a
+      className={className}
+      rel="noopener noreferrer"
+      target="_blank"
+      href={`http${decode ? '' : 's'}://${decode ? decodeHex(domain) : domain}`}
```

Comment:
Always use `https`.  I think this was a bug that some places did it while others did not.